docs/installation.md

```diff
@@ -160,37 +189,43 @@ cd ..
 rm -rf ./ta-lib*
 ```
 
-!!! Note
-    An already downloaded version of ta-lib is included in the repository, as the sourceforge.net source seems to have problems frequently.
 
-#### 2. Setup your Python virtual environment (virtualenv)
+#### Install Freqtrade
 
-!!! Note
-    This step is optional but strongly recommended to keep your system organized
+Clone the git repository:
 
 ```bash
-python3 -m venv .env
-source .env/bin/activate
+# download repository
+git clone https://github.com/freqtrade/freqtrade.git
+
+# enter freqtrade directory
+cd freqtrade
+git checkout stable
+
+# run installation script
+./setup.sh --install
```

Comment:
This guide explains 3 ways to install Freqtrade now.
* Quick installation with `./setup.sh`
* Manual / custom installation without `./setup.sh`
* Conda

Since we're in the "custom" part - this is misplaced in this location - as by reading to this point, the user already decided to not use the setup.sh install script.

Reply:
Sorry but NO,
Quick installation, in fact explains nothing, because user has to go to Manual installation, to build prerequisites, and because of that- it is the same. so the document explain 2 ways + point to docker

Reply:
Then you didn't understand the quick install script.

the ONLY dependency that the quick install script requires is python, pip and venv.
Ta-lib is installed by the install script (on linux and mac, the script is not compatible with windows).

Reply:
Perhaps.

 However, it is interesting that the original manual does not explain or point out why there are two versions of the installation when both use the /setup.sh script.

Please show then how you would like to see the `installation.md`

Reply:
> However, it is interesting that the original manual does not explain or point out why there are two versions of the installation when both use the /setup.sh script.

This is not true.
If you go to the [existing documentation](https://www.freqtrade.io/en/latest/installation/#custom-installation) you'll find that `setup.sh` is not mentioned anywhere below the "custom installation" header.

And that's what i mean - we have setup.sh - which is the quick installation script (which works for macos and debian based linux).
If you don't use that (say you use centos) - you MUST use the "custom installation" - which therefore is also required.